### DISCUSSION TITLE
Missing Yearly View in Species Stats

### DISCUSSION THREAD

**Discussion by DD4WH** [mcguirepr89/BirdNET-Pi#913] [Q&A]
In my older installations, I was able to setup the Yearly View by adding a sunrise table to the SQLite databases and doing some more steps, so that I could then choose between different Graphs. This has never been a standard option, although I think it should.

With a new installation, I tried to setup the Yearly view. However, I miss the `sun.py` script. I use this information:

  _Originally posted by @CaiusX in https://github.com/mcguirepr89/BirdNET-Pi/discussions/470#discussioncomment-3273274_

Can anybody give me a hint on how to activate the yearly view, which is really the most informative graph that BirdNET-Pi can produce?

**Comment by ehpersonal38** [comment]
On the left hand side of species stats there's a resample resolution button called "DAILY" I think. Click that and it should give you the yearly view complete with sunrise and sunset times.

**Reply by DD4WH** [reply]
Thanks Ethan for your answer!

No, if I click on single day view (which is the only option to click), there is no yearly view, but I can choose a single day for the graph.

**Reply by DD4WH** [reply]
Oh, sorry, I did not read properly! Clicking "Daily" gives the missing yearly view, thanks a lot!!!!
